packages/examples/src/model/client.ts

```diff
@@ -0,0 +1,15 @@
+/* --------------------------------------------------------------------------------------------
```

Comment:
Please move it to `language-client-runner.ts`. I don't see the need for this artificial separation.

Reply:
Please move to `client-commons.ts`. The model sub-folder is not needed now.
I overlooked the circular dependency. I want to think about how to organize/move the common code out of the examples after this PR is merged.